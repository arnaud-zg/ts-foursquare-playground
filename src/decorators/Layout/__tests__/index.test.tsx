import React from 'react'
import renderer from 'react-test-renderer'
import { Layout } from '..'

describe('Component/ResultState', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Layout>
          <div>Custom content</div>
        </Layout>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
