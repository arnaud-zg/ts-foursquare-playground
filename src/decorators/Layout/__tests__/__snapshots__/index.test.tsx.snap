// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/ResultState renders correctly 1`] = `
Array [
  <div
    className="bg-gray-400 w-11/12 p-2 mx-auto mt-2 rounded-t text-center"
  >
    <h1
      className="font-sans font-thin text-4xl"
    >
      "ts-foursquare" is just a simple library
    </h1>
  </div>,
  <body
    className="mx-auto w-11/12"
  >
    <div>
      Custom content
    </div>
  </body>,
]
`;
